config-connector/solutions/iam/kpt/pubsub-subscription/README.md

```diff
@@ -0,0 +1,41 @@
+Pub/Sub Subscription
+==================================================
+# NAME
+  pubsub-subscription
+# SYNOPSIS
+  This package configures permissions for a pubsub subscription by creating an IAMPolicyMember resource
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/):
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/pubsub-subscription pubsub-subscription
+  ```
+# REQUIREMENTS
+
+  A Config Connector installation managing a GCP project with Pub/Sub API enabled. 
+
+# USAGE
+## SETTERS
+|       NAME        |        VALUE         |     SET BY      |          DESCRIPTION           | COUNT |
+|-------------------|----------------------|-----------------|--------------------------------|-------|
+| iam-member        | ${IAM_MEMBER?}       | PLACEHOLDER     | IAM member to grant role       | 1     |
+| role              | roles/pubsub.viewer  | package-default | IAM role to grant              | 1     |
+| subscription-name | allowed-subscription | package-default | name of PubSub subscription    | 2     |
+| topic-name        | allowed-topic        | package-default | name of PubSub topic           | 2     |
+
```

Comment:
Maybe add another title ## BASIC WORKFLOW under it? Just having one subtitle seems to be confusing.

Reply:
I have moved this table into its own section instead.

Reply:
We've discussed offline and agreed that we'll move the setter table out of the usage section.